### PR TITLE
Rchain 3118: remove MakePos.rho

### PR DESCRIPTION
**Pull request by odeac**
## Overview
see title

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3318

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by KentShikama** [comment]
Is this based on another PR?

**Comment by odeac** [comment]
yes. I thought the other two are merged by the time this is reviewed but I got lint problems. I'm fixing them asap

**Comment by KentShikama** [comment]
bors try

**Comment by odeac** [comment]
bors try

File: casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala [hunk 1]
```diff
@@ -155,16 +155,18 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
...
     s"""
-       | new rl(`rho:registry:lookup`), SystemInstancesCh, posCh in {
-       |   rl!(`rho:rchain:systemInstancesRegistry`, *SystemInstancesCh) |
-       |   for(@(_, SystemInstancesRegistry) <- SystemInstancesCh) {
-       |     @SystemInstancesRegistry!("lookup", "pos", *posCh) |
-       |     for(pos <- posCh){ pos!("getBonds", "$name") }
+       | new rl(`rho:registry:lookup`), objCh in {
+       |   rl!(`$uri`, *objCh) |
+       |   for(@(_, obj) <- objCh) {
+       |     @obj!(${args.map(_.toString).mkString(",")})
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: Please revert this, this obscures what rholang is being executed and prevents plugging in debugging. I'm surprised to be saying this, but not all deduplication makes sense.

Comment (odeac) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: yeah, I agree. It was an experiment I did and I forgot to undo

File: casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala [hunk 2]
```diff
@@ -202,13 +201,13 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
...
-       |   }
-       | }
-       """.stripMargin
+       # new rl(`rho:registry:lookup`), poSCh in {
+       #   rl!(`rho:rchain:pos`, *poSCh) |
+       #   for(@(_, PoS) <- poSCh) {
+       #     @PoS!("pay", $amount, "$name")
+       #   }
+       # }
+       """.stripMargin('#')
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: really, this should've been a separate commit. It's even a different file, I don't understand why you didn't split this out.

**Comment by odeac** [comment]
bors try

**Comment by odeac** [comment]
bors try

**Pull request merged**